region/wreckedship/main/Bowling Alley.json

```diff
@@ -428,9 +422,9 @@
       "notable": true,
       "requires": [
         "h_canCeilingBombJump",
+        "canLongCeilingBombJump",
         "canBeVeryPatient"
```

Comment:
```suggestion
        "canBeExtremelyPatient"
```

Reply:
Looks longer than the fire flea CIBJ

Reply:
I'm on the fence about this one. It will be in `Beyond` either way, but this one should only take 4 minutes (according to your notable video).

Reply:
I didn't think I ran that CIBJ to the end.  I like very patient then.  My setup could have been done a little faster whereas in firefleas you have to figure out how to kill the Wavers.